### PR TITLE
Maboelsoudd2l/add action toast

### PR DESCRIPTION
**Pull request by maboelsoudd2l**
pretty simple

### CODE REVIEW

File: components/d2l-quick-eval/d2l-quick-eval-activities.js [hunk 1]
```diff
@@ -450,6 +452,10 @@ class D2LQuickEvalActivities extends mixinBehaviors(
 				const selfHref = this._getSelfLink(this.entity);
 				// bypass cache and reload
 				window.D2L.Siren.EntityStore.fetch(selfHref, this.token, true);
+				this.shadowRoot.querySelector('#toast-dismiss-success').open = true;
```
Comment (AlexBedley) [inline components/d2l-quick-eval/d2l-quick-eval-activities.js]: If I dismiss activity A, then dismiss activity B, will I see this toast after dismissing activity B?

File: components/d2l-quick-eval/d2l-quick-eval-activities.js [hunk 2]
```diff
@@ -450,6 +452,10 @@ class D2LQuickEvalActivities extends mixinBehaviors(
...
+			}).catch((error) => {
+				this.shadowRoot.querySelector('#toast-dismiss-critical').open = true;
+				throw new Error(`could not dismiss activity, error: ${error}`);
```
Comment (AlexBedley) [inline components/d2l-quick-eval/d2l-quick-eval-activities.js]: won't this cause the app to display the global error message and freeze? I think in this case we'd like the non-dismiss stuff to keep working, but happy to defer to Dan

File: components/d2l-quick-eval/d2l-quick-eval-activities.js [hunk 3]
```diff
@@ -183,6 +183,8 @@ class D2LQuickEvalActivities extends mixinBehaviors(
 				<d2l-button slot="footer" dialog-action="no">[[localize('no')]]</d2l-button>
 			</d2l-dialog-confirm>
 			<d2l-quick-eval-action-dismiss-dialog></d2l-quick-eval-action-dismiss-dialog>
+			<d2l-alert-toast id="toast-dismiss-success" type="success">Activity Dismissed</d2l-alert-toast>
+			<d2l-alert-toast id="toast-dismiss-critical" type="critical">Activity could not be dismissed. Please try again.</d2l-alert-toast>
```
Comment (AlexBedley) [inline components/d2l-quick-eval/d2l-quick-eval-activities.js]: I think at this point we should be adding langterms

File: components/d2l-quick-eval/d2l-quick-eval-activities.js (hunk 1 above)
Comment (maboelsoudd2l) [reply components/d2l-quick-eval/d2l-quick-eval-activities.js]: yeah it auto-closes, so you have to keep doing this everytime

File: components/d2l-quick-eval/d2l-quick-eval-activities.js (hunk 2 above)
Comment (maboelsoudd2l) [reply components/d2l-quick-eval/d2l-quick-eval-activities.js]: what would you like instead?

Comment (AlexBedley) [reply components/d2l-quick-eval/d2l-quick-eval-activities.js]: chatted offline, gonna log using the logging service

File: components/d2l-quick-eval/d2l-quick-eval-activities.js (hunk 3 above)
Comment (maboelsoudd2l) [reply components/d2l-quick-eval/d2l-quick-eval-activities.js]: cool

File: components/d2l-quick-eval/d2l-quick-eval-activities.js (hunk 1 above)
Comment (AlexBedley) [reply components/d2l-quick-eval/d2l-quick-eval-activities.js]: I could not reproduce this. have you tried it?

**Pull request merged**